user_guide/data_management_data_audit.rst

```diff
@@ -3,8 +3,18 @@
 Data Audit
 ==========
 
-To see an out-of-the-box report of all the actions, performed with records of an
-:ref:`auditable entity <user-guide-entity-management-create-other>`, go to *System → Data Audit*.
+With the functionality, the user can see full history of changes made to any record of an 
+:ref:`auditable entity <user-guide-entity-management-create-other>` and the out-of-the-box 
+report of all such actions.
+
+.. hint::
+
+    The set of entities, for which data audit is defined by the system administrator. Any entity can be 
+    :ref:`set as auditable <user-guide-entity-management-create-other>`.
```

Comment:
Comment: The first sentence ("The set of entities...") is a fragment, and so is difficult to edit. What is a set of entities? Can this be clarified?